### PR TITLE
Add the 1.2.11 Release Note

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the 1.2.11 release note.

> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: downloads/release-notes/1.2.11/RELEASE_NOTE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: ballerina-blank-page
+title: Release Note
+---
+### Overview of jBallerina 1.2.11
+
+The jBallerina 1.2.11 patch release improves upon the 1.2.10 release by addressing a number of [bugs](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A%22Ballerina+1.2.11%22+label%3AType%2FBug+is%3Aclosed).
```
Comment (ThisaruGuruge) [inline downloads/release-notes/1.2.11/RELEASE_NOTE.md:7]:
```suggestion
The jBallerina 1.2.11 patch release improves upon the 1.2.10 release by addressing a number of [issues](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A%22Ballerina+1.2.11%22+is%3Aclosed).
```

File: downloads/release-notes/1.2.11/RELEASE_NOTE.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+**For new users:**
+
+If you have not installed jBallerina, then download the [installers](/downloads/) to install.
+
+#### Standard Library
+
+##### HTTP
+
+Introduce HTTP 1.1 client inbound response validation.
```
Comment (chamil321) [inline downloads/release-notes/1.2.11/RELEASE_NOTE.md:30]:
```suggestion
Introduce inbound response validation for HTTP/1.1 client. 
```

**Review by chamil321 — APPROVED** [review]

File: downloads/release-notes/1.2.11/RELEASE_NOTE.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Introduce inbound response validation for HTTP/1.1 client. 
+
+- `maxStatusLineLength` - The maximum length allowed for the response status line. Exceeding this limit will result in an error.
+- `maxHeaderSize` - The maximum size allowed for headers. Exceeding this limit will result in an error.
+- `maxEntityBodySize` - The maximum size allowed for the entity body. By default, it is set to -1, which means there is no restriction on the`maxEntityBodySize`. Exceeding this limit will result in an error.
+
+##### Cache
+
+Improve the concurrent behavior of the cache.
```
Comment (ThisaruGuruge) [inline downloads/release-notes/1.2.11/RELEASE_NOTE.md:38]:
```suggestion
- Improve the concurrent behavior of the cache.
```

Comment (praneesha) [reply downloads/release-notes/1.2.11/RELEASE_NOTE.md:38]: We don't need bullets for this as we don't have multiple points under this topic.

Comment (ThisaruGuruge) [reply downloads/release-notes/1.2.11/RELEASE_NOTE.md:38]: Okay.

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**